oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/storage/EBPFProfilingTaskRecord.java

```diff
@@ -41,30 +40,20 @@
 @Stream(name = EBPFProfilingTaskRecord.INDEX_NAME, scopeId = EBPF_PROFILING_TASK,
         builder = EBPFProfilingTaskRecord.Builder.class, processor = NoneStreamProcessor.class)
 public class EBPFProfilingTaskRecord extends NoneStream {
-
     public static final String INDEX_NAME = "ebpf_profiling_task";
-    public static final String PROCESS_FIND_TYPE = "process_find_type";
     public static final String SERVICE_ID = "service_id";
-    public static final String INSTANCE_ID = "instance_id";
-    public static final String PROCESS_ID = "process_id";
-    public static final String PROCESS_NAME = "process_name";
+    public static final String PROCESS_LABELS_JSON = "process_labels_json";
     public static final String START_TIME = "start_time";
     public static final String TRIGGER_TYPE = "trigger_type";
     public static final String FIXED_TRIGGER_DURATION = "fixed_trigger_duration";
     public static final String TARGET_TYPE = "target_type";
     public static final String CREATE_TIME = "create_time";
     public static final String LAST_UPDATE_TIME = "last_update_time";
 
-    @Column(columnName = PROCESS_FIND_TYPE)
-    private int processFindType = EBPFProfilingProcessFinderType.UNKNOWN.value();
     @Column(columnName = SERVICE_ID)
     private String serviceId;
-    @Column(columnName = INSTANCE_ID, length = 600)
-    private String instanceId;
-    @Column(columnName = PROCESS_ID, length = 600)
-    private String processId;
-    @Column(columnName = PROCESS_NAME, length = 500)
-    private String processName;
+    @Column(columnName = PROCESS_LABELS_JSON)
```

Comment:
I think we should have the length of the label?

Reply:
updated.